services/newsfeed/tests/functional/mysql.test.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const mysqlIndex = require('../../shared/indexdb/mysql');
+const testschema = require('../../shared/schema/newsfeed');
+const {
+	invalidSchemaWithMissingType,
+	invalidSchemaWithoutPrimaryKey,
+} = require('../schemas/invalidSchemas');
+
+const getIndex = () => mysqlIndex('testSchemaNewsfeed', testschema);
+
+const { news, drupalData } = require('../constants/newsfeed');
+
+describe('Test mysql', () => {
+	let db;
+	beforeAll(async () => {
+		// Create table
+		db = await getIndex();
+	});
+
+	afterAll(async () => {
+		// TODO: Drop table
+		const result = await db.find();
+		await db.deleteIds(result.map(r => r[`${testschema.primaryKey}`]));
+	});
```

Comment:
Please drop the test table after all the tests have finished.
```suggestion
const tableName = 'testSchemaNewsfeed';	
const getIndex = () => mysqlIndex(tableName, testschema);

const { news, drupalData } = require('../constants/newsfeed');

describe('Test mysql', () => {
	let db;
	beforeAll(async () => {
		// Create table
		db = await getIndex();
	});

	afterAll(async () => testDB.rawQuery(`DROP TABLE IF EXISTS ${tableName} CASCADE`));
```